### DISCUSSION TITLE
The payment of the block won in wallet does not appear

### DISCUSSION THREAD

**Discussion by Coinff01** [Chia-Network/chia-blockchain#8382] [Support]
My key to receive is: xch1ydk6rygs7ev4hfyvq55qhvf9p0kea4fhzqfktqsl9qjasgd56a5q5lduvu

Key that always received the rewards from the pool, mojo from https://faucet.chia.net/

But he dreamed

Win a block with the hash: 0x0xc4f710997d042b6b262b69ae7f75ba7fd9fe101b05d5bf417f8638455b9667d5

Then the key to my farm showed a message that said "one or more addresses do not have private keys", the key that I always use, so far I have not received the reward nor does the farm acknowledge that I found any blocks.

But the block I win appears in the pool and when I look for it, my receiving address appears.

I proceeded to change it but nothing, it never receives the block reward (0.25) for a strange problem in my key

![Key](https://user-images.githubusercontent.com/89061520/132424186-440c9202-29d4-4c80-9002-fa40e19df597.jpeg)

![block](https://user-images.githubusercontent.com/89061520/132424066-6aa58149-84f0-4ce9-b1cc-a491fead2276.png)

![image](https://user-images.githubusercontent.com/89061520/132424228-76ecbf5d-7c27-46c6-8397-a8419aac0d6c.png)

the only way you would stop having control of the same address that you have used previously, if you removed a key from Chia - did you have other keys previously that you have removed ?

It was not the key that I copied and always used, until that situation

**Comment by jack60612** [comment]
> My key to receive is: xch1ydk6rygs7ev4hfyvq55qhvf9p0kea4fhzqfktqsl9qjasgd56a5q5lduvu
> 
> Key that always received the rewards from the pool, mojo from https://faucet.chia.net/
> 
> But he dreamed
> 
> Win a block with the hash: 0x0xc4f710997d042b6b262b69ae7f75ba7fd9fe101b05d5bf417f8638455b9667d5
> 
> Then the key to my farm showed a message that said "one or more addresses do not have private keys", the key that I always use, so far I have not received the reward nor does the farm acknowledge that I found any blocks.
> 
> But the block I win appears in the pool and when I look for it, my receiving address appears.
> 
> I proceeded to change it but nothing, it never receives the block reward for a strange problem in my key
> 
> ![Key](https://user-images.githubusercontent.com/89061520/132424186-440c9202-29d4-4c80-9002-fa40e19df597.jpeg)
> 
> ![block](https://user-images.githubusercontent.com/89061520/132424066-6aa58149-84f0-4ce9-b1cc-a491fead2276.png)
> 
> ![image](https://user-images.githubusercontent.com/89061520/132424228-76ecbf5d-7c27-46c6-8397-a8419aac0d6c.png)

Is your wallet synced?

**Reply by Coinff01** [reply]
Yes, my wallet synced

**Reply by jack60612** [reply]
Wait you do realize that your in a pool so you don't get a reward

**Reply by Coinff01** [reply]
You're wrong. 0.25 corresponds to me, 1.75 for the pool.

**Reply by jack60612** [reply]
@Coinff01 you don't get the 0.25 as it goes to another address that your client can't access as shown by "one or more addresses do not have private keys" You don't get the 1.75 either as you get the pool reward instead

**Reply by Coinff01** [reply]
That is the problem, that address was the one I always used and at the moment it is no longer mine, as it happens, if via that address everything reached me, as a change if it was the one I took from the chia client.

AND THANK YOU FOR YOUR TIME

**Reply by Coinff01** [reply]
Farmers trust in this system, how can we change our receiving addresses just when we win a block, how is it possible that my usual address, where I receive from the pool and even from the jaw, changes at the moment and that is no longer my password since I do not have private keys when my client generated it and my farm has used it, very strange and delicate this

**Reply by loppefaaret** [reply]
the only way you would stop having control of the same address that you have used previously, if you removed a key from Chia - did you have other keys previously that you have removed ?

**Comment by BrandtH22** [comment]
@Coinff01 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
